examples/src/bin/listbox_model.rs

```diff
@@ -365,17 +333,42 @@ mod row_data {
         // This maps between the GObject properties and our internal storage of the
         // corresponding values of the properties.
         impl ObjectImpl for RowData {
-            fn set_property(&self, _obj: &Self::Type, id: usize, value: &glib::Value) {
-                let prop = &PROPERTIES[id];
+            fn properties() -> Vec<glib::ParamSpec> {
```

Comment:
Does it has to return a Vec here?

Reply:
What else?

Reply:
What I meant is whether it can return a `&'static [glib::ParamSpec]` ? :)

Reply:
No because creating a paramspec is not a const function. You wouldn't be able to initialize that array with anything.

Reply:
Well, you could go with storing it in a `Lazy` inside the function and returning it from that... but that seems overly complicated but arguably more correct.

Would you prefer that?

Reply:
Well, let's change it to that. Then at least it gives reliable results if called multiple times

Reply:
>  Well, you could go with storing it in a Lazy inside the function and returning it from that... but that seems overly complicated but arguably more correct.

Well, if the plan is to provide some nice macro around all of this, I would rather go for correctness as it's already a bit complex to do all the properties stuff

Reply:
This is implemented in https://github.com/gtk-rs/gtk-rs/pull/270 now, which supersedes this PR.

I'll update this PR to only include the `Value` changes once the other one is merged.